tests/smoke.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+TESTS="pwd, lsb_release -a, uname -a, ip a, lscpu, vmstat, lsblk, false"
+LOGS="/tmp/TESTS_$(date +%s)_"
+
+[ -z $TEST_DIR ] && (echo "ERROR: TEST_DIR not defined"; exit 1)
+[ -d $TEST_DIR ] || (echo "ERROR: no TEST_DIR"; exit 1)
+
+run() {
+    # shellcheck disable=SC2039
+    local test="$1"
+    test_case_id="$(echo "${test}" | awk '{print $1}')"
+	result_dir="${TEST_DIR}/$(date +%s.%N)-${test_case_id}"
+	mkdir ${result_dir}
+    echo
+    echo "Running ${test_case_id} test..."
+    local exit_code=0
+    eval "${test}" || exit_code="$?"
```

Comment:
this is totally subjective, but our ci-scripts uses something like this to help group the output of subcommands:
~~~
indent() { 
    sed 's/^/|  /'
    echo "|--" 
}

# then you could do:
eval "${test}" | indent || exit_code="$?"
~~~

Which produces output like:
~~~
Running lsblk test...
|  NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
|  mmcblk0     179:0    0 29.8G  0 disk 
|  |-mmcblk0p1 179:1    0 41.5M  0 part /var/rootdirs/mnt/boot
|  `-mmcblk0p2 179:2    0 29.8G  0 part /sysroot
|--
Exit code: 0
Return code: 0
~~~